azure-pipelines.yml

```diff
@@ -200,6 +206,8 @@ stages:
   parameters:
     backwardCompatibleRelease: $(backwardCompatibleRelease)
     forwardCompatibleRelease: $(forwardCompatibleRelease)
+    enableForwardCompatibleTests: true
+    enableBackwardCompatibleTests: true
```

Comment:
Shouldn't these be part of the `tests` parameter ?

Reply:
I don't think there's an easy way to add default value for this parameter.  May have to readd them back to each of the `tests`.

Reply:
Oops you're right, fixed.